commit be26db61e880b3892f189e9ef54b7b80599245bf
Author: Oliver Upton <oliver.upton@linux.dev>
Date:   Mon Apr 22 20:01:55 2024 +0000

    KVM: selftests: Add a minimal library for interacting with an ITS
    
    A prerequisite of testing LPI injection performance is of course
    instantiating an ITS for the guest. Add a small library for creating an
    ITS and interacting with it from the guest.
    
    Signed-off-by: Oliver Upton <oliver.upton@linux.dev>
    Link: https://lore.kernel.org/r/20240422200158.2606761-17-oliver.upton@linux.dev
    Signed-off-by: Marc Zyngier <maz@kernel.org>

diff --git a/tools/testing/selftests/kvm/include/aarch64/vgic.h b/tools/testing/selftests/kvm/include/aarch64/vgic.h
index ce19aa0a8360..c481d0c00a5d 100644
--- a/tools/testing/selftests/kvm/include/aarch64/vgic.h
+++ b/tools/testing/selftests/kvm/include/aarch64/vgic.h
@@ -32,4 +32,6 @@ void kvm_irq_write_isactiver(int gic_fd, uint32_t intid, struct kvm_vcpu *vcpu);
 
 #define KVM_IRQCHIP_NUM_PINS	(1020 - 32)
 
+int vgic_its_setup(struct kvm_vm *vm);
+
 #endif // SELFTEST_KVM_VGIC_H